### ISSUE TITLE
Improved export/dialog dialog

### ISSUE THREAD

**Issue by mejo-** [nextcloud/collectives#463]
The idea is to transform the print view into a modal with different export options and a print option. The loading feedback while export is generated can be similar to the current splash screen of the print view.

**Comment by mejo-** [comment]
Thanks @thopanX. Can you elaborate what you mean with header/footer? Do you mean the title of the page in question?

**Comment by mejo-** [comment]
Related to #469

**Comment by max-nextcloud** [comment]
@thopanX I think that header is added by the print dialogue of your browser and can be turned off. (See the blue checkmark below):

![grafik](https://user-images.githubusercontent.com/97337118/209465613-aaa76dc2-5590-413f-ace7-715022422309.png)
(`more settings` in print dialogue of firefox)

I'm not sure if we can make it configurable because it mainly consistes of the page number, date and url. So things that we cannot change easily.

**Comment by max-nextcloud** [comment]
>> I'm not sure if we can make it configurable because it mainly consistes of the page number, date and url. So things that we cannot change easily.

> But what can be improved would be the name of the export, right? It currently doesn't mention the name of the collective, just „Kollektive – Nextcloud“.

Yes. I think that is the page title. That should be fairly simple actually.

**Comment by max-nextcloud** [comment]
Related to #375
